src/controllers/main/main.ts

```diff
@@ -426,7 +427,7 @@ export class MainController extends EventEmitter {
     const state = this.accounts.accountStates[accountOp.accountAddr][accountOp.networkId]
     const provider = this.providers.providers[network.id]
     const gasPrice = this.gasPrices[network.id]
-    const addresses = await debugTraceCall(
+    const { tokens, nfts } = await debugTraceCall(
```

Comment:
we don't have any tests for `debugTraceCall` lib nor for the`main->traceCall`, would be great if we write tests for the discovery mechanism